BitmovinConvivaAnalytics/Classes/ConvivaAnalytics.swift

```diff
@@ -181,14 +172,14 @@ public final class ConvivaAnalytics: NSObject {
             return
         }
 
-        if player.config.sourceItem?.itemTitle == nil && contentMetadataBuilder.assetName == nil {
+        if player.config.sourceItem?.itemTitle == nil, contentMetadataBuilder.assetName == nil {
             throw ConvivaAnalyticsError(
                 // swiftlint:disable:next line_length
                 "AssetName is missing. Load player source (with itemTitle) first or set assetName via updateContentMetadata"
             )
         }
 
-        internalInitializeSession()
+        internalInitializeSession(fromPlay: true)
```

Comment:
This is not really coming `fromPlay` so I don't really like the parameter name 🤔